lib/api/v1/forms/sign_up_test.go

```diff
@@ -50,4 +51,61 @@ var _ = Describe("SignUp", func() {
 			})
 		})
 	})
+
+	Describe("Validations", func() {
+		Context("Given INVALID email", func() {
+			It("returns the error", func() {
+				form := forms.SignUpForm{
+					Email:                "INVALID",
+					Password:             "12345678",
+					PasswordConfirmation: "12345678",
+				}
+
+				err := validators.Validate(form)
+
+				Expect(err).ToNot(BeNil())
+			})
+		})
+
+		Context("Given EMPTY password", func() {
+			It("returns the error", func() {
+				form := forms.SignUpForm{
+					Email:                "example@gmail.com",
+					PasswordConfirmation: "12345678",
+				}
+
+				err := validators.Validate(form)
+
+				Expect(err).ToNot(BeNil())
+			})
+		})
+
+		Context("Given password less than min requirement", func() {
+			It("returns the error", func() {
+				form := forms.SignUpForm{
+					Email:                "example@gmail.com",
+					Password:             "123",
+					PasswordConfirmation: "123",
+				}
+
+				err := validators.Validate(form)
+
+				Expect(err).ToNot(BeNil())
+			})
+		})
+
+		Context("Given INVALID password confirmation", func() {
+			It("returns the error", func() {
```

Comment:
In that case, as the error has no identity (e.g. we don't know if it's an "invalid" or "blank" or "confirmed" error), it would be "an" (`it returns an error`).

Unless we test for a specific error (e.g. `it returns the confirmation error` and `expect err to eq(ConfirmationError)`).